.github/codecov.yml

```diff
@@ -0,0 +1,12 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: 0.5%
+
+github_checks:
+  annotations: false
+
+ignore:
+  - "setup.py"
+  - "src/qibo/tests/test_custom_functions.py"
```

Comment:
I know it existed before this PR, but I have the impression that these files do not exist anymore, so we don't need these lines.

Reply:
You're right, I simply ignored the issue. But it costs little, so it's worth to do it.

Solved in 47b2518